documentation/api/io.strimzi.api.kafka.model.kafka.KafkaClusterSpec.adoc

```diff
@@ -119,30 +119,21 @@ Be aware that if the configured image is not compatible with Strimzi images, it
 
 Kafka has its own configurable loggers, which include the following:
 
-* `log4j.logger.kafka`
-* `log4j.logger.org.apache.kafka`
-* `log4j.logger.kafka.request.logger`
-* `log4j.logger.kafka.network.Processor`
-* `log4j.logger.kafka.server.KafkaApis`
-* `log4j.logger.kafka.network.RequestChannel$`
-* `log4j.logger.kafka.controller`
-* `log4j.logger.kafka.log.LogCleaner`
-* `log4j.logger.state.change.logger`
-* `log4j.logger.kafka.authorizer.logger`
-
-Kafka uses the Apache `log4j` logger implementation.
-
+* `logger.kafka`
+* `logger.org.apache.kafka`
+* `logger.kafka.request.logger`
+* `logger.kafka.network.RequestChannel$`
+* `logger.org.apache.kafka.controller`
+* `logger.kafka.log.LogCleaner`
+* `logger.state.change.logger`
+* `logger.kafka.authorizer.logger`
```

Comment:
I think this is not correct. See my previous comments and also the examples included in the following PR for each major component.

https://github.com/strimzi/strimzi-kafka-operator/pull/11176/files#diff-96dbf0bf20d7d83f0a33459b088dd6223e3c46f114c989e177c335505369d9f6